Tests/Command/RemoveThumbsCommandTest.php

```diff
@@ -0,0 +1,148 @@
+
+        $this->pool = $pool = $this->getMockBuilder('Sonata\MediaBundle\Provider\Pool')->disableOriginalConstructor()->getMock();
+
+        $this->mediaManager = $mediaManager = $this->getMockBuilder('Sonata\MediaBundle\Model\MediaManagerInterface')->getMock();
+
+        $this->container->expects($this->any())
+            ->method('get')
+            ->will($this->returnCallback(function ($id) use ($pool, $mediaManager) {
+                switch ($id) {
+                    case 'sonata.media.pool':
+                        return $pool;
+                    case 'sonata.media.manager.media':
+                        return $mediaManager;
+                }
+
+                return;
+            }));
+    }
+
+    public function testExecuteWithoutArguments()
+    {
+        $this->filesystem->mkdir($this->workspace.DIRECTORY_SEPARATOR.'foo');
+        $this->filesystem->touch($this->workspace.DIRECTORY_SEPARATOR.'foo'.DIRECTORY_SEPARATOR.'thumb_1_foo.ext');
+        $this->filesystem->touch($this->workspace.DIRECTORY_SEPARATOR.'foo'.DIRECTORY_SEPARATOR.'thumb_2_bar.ext');
+
+        $context = [
+            'providers' => [],
+            'formats' => [],
+            'download' => [],
+        ];
+
+        $formats = [
+            'small' => [],
+        ];
+
+        $filesystem = $this->getMockBuilder('Gaufrette\Filesystem')->disableOriginalConstructor()->getMock();
+        $fileProvider = $this->getMockBuilder('Sonata\MediaBundle\Provider\FileProvider')->disableOriginalConstructor()->getMock();
+        $fileProvider->expects($this->any())->method('getName')->will($this->returnValue('fooprovider'));
+        $fileProvider->expects($this->once())->method('getFormats')->will($this->returnValue($formats));
+        $fileProvider->expects($this->exactly(2))->method('getFilesystem')->will($this->returnValue($filesystem));
+        $fileProvider->expects($this->exactly(2))->method('removeThumbnails');
+
+        $this->pool->expects($this->once())->method('getContexts')->will($this->returnValue(['foo' => $context]));
+        $this->pool->expects($this->once())->method('getProviders')->will($this->returnValue(['fooprovider' => $fileProvider]));
+        $this->pool->expects($this->once())->method('getProvider')->will($this->returnValue($fileProvider));
+
+        $medias = [];
+
+        $media = new Media();
+        $media->setId(1);
+        $media->setName('foo');
+        $medias[] = $media;
+
+        $media = new Media();
+        $media->setId(2);
+        $media->setName('bar');
+        $medias[] = $media;
+
+        $this->mediaManager->expects($this->exactly(2))->method('findBy')->will($this->returnCallback(function (array $criteria, array $orderBy = null, $limit = null, $offset = null) use ($medias) {
+            if (null !== $offset && $offset > 0) {
+                return [];
+            }
+
+            return $medias;
+        }));
+
+        $this->tester->setInputs(['fooprovider', 'foo', 'small']);
```

Comment:
This line is making travis fail, can you take a look at it?

Reply:
I think there is another problem:
> TypeError: Return value of Sonata\MediaBundle\Command\RemoveThumbsCommand::getQuestionHelper() must be an instance of Sensio\Bundle\GeneratorBundle\Command\Helper\QuestionHelper, instance of Symfony\Component\Console\Helper\QuestionHelper returned

So all this time we were accessing a different helper, when we were calling `$this->getHelper('question')`.